src/frontend/Komponenter/Personoversikt/Infotrygdperioderoversikt.tsx

```diff
@@ -77,42 +67,20 @@ const InfotrygdEllerSummertePerioder: React.FC<{ perioder: InfotrygdPerioderResp
                         checked={visSummert}
                     />
                 )}
-                <Checkbox
-                    label={'Større tabeller'}
-                    onChange={() => {
-                        settVisStørreTabell((prevState) => !prevState);
-                    }}
-                    checked={visStørreTabell}
-                />
             </CheckboxContainer>
             <h2>Overgangsstønad</h2>
-            {visPerioder(
-                Stønadstype.OVERGANGSSTØNAD,
-                visSummert,
-                visStørreTabell,
-                perioder.overgangsstønad
-            )}
+            {visPerioder(Stønadstype.OVERGANGSSTØNAD, visSummert, perioder.overgangsstønad)}
 
             <h2>Barnetilsyn</h2>
-            {visPerioder(
-                Stønadstype.BARNETILSYN,
-                visSummert,
-                visStørreTabell,
-                perioder.barnetilsyn
-            )}
+            {visPerioder(Stønadstype.BARNETILSYN, visSummert, perioder.barnetilsyn)}
 
             <h2>Skolepenger</h2>
-            {visPerioder(
-                Stønadstype.SKOLEPENGER,
-                visSummert,
-                visStørreTabell,
-                perioder.skolepenger
-            )}
+            {visPerioder(Stønadstype.SKOLEPENGER, visSummert, perioder.skolepenger)}
         </>
     );
 };
 
-const Infotrygdperioderoversikt: React.FC<{
+export const Infotrygdperioderoversikt: React.FC<{
```

Comment:
Er det noen spesiell grunn til at denne ble endret til `export const` i stedet for å bruke `export default`?

Reply:
Sannsynligvis ikke. Eneste forskjellen er at nå må man importere slik:
`import { Infotrygdperioderoversikt } from "path/Infotrygdperioderoversikt";`

Dersom man har default export blir importeringen marginalt mer konsis:
`import Infotrygdperioderoversikt from "path/Infotrygdperioderoversikt";`

Vi har ingen spesielle regler på om vi skal bruke default eller Named export